Game/Source/GDKTestGymsFunctionalTests/GDKTestGymsFunctionalTests.cpp

```diff
@@ -2,8 +2,20 @@
 
 #include "GDKTestGymsFunctionalTests.h"
 #include "LogGymsSpatialFunctionalTest.h"
-#include "Modules/ModuleManager.h"
+#include "TestMapGeneration.h"
```

Comment:
Is this include okay? No module dependency issues?
Either way, I think @MatthewSandfordImprobable would know best: will this work in the NFRs?
When we are building e.g. the Client or Server build target.